### ISSUE TITLE
Unsafe remove button from Project> Settings> Members

### ISSUE THREAD

**Issue by b14cknc0d3** [appwrite/appwrite#3626]
### 👟 Reproduction steps

One of the bad experiences I got while using appwrite is there is a little bug in. Project> Settings> Member s
I am testing my email setup from the member invite     
, and there are three owners .and I accidentally pressed the remove button of the first owner and even don't show dialogue to confirm
(i repeatedly removed another member too ) and the whole project is gone while there are still two owners.

### 👍 Expected behavior

There must be  **type a name to remove owner** and **warning** about if you remove that owner the entire project will be deleted.

### 👎 Actual Behavior

no warning,unsafe

### 🎲 Appwrite version

Version 0.15.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by manishkumar-dsi** [comment]
@stnguyen90 Please assign to me. I would like to work on this issue.

**Comment by stnguyen90** [comment]
@manishkumar-dsi, assigned! Thanks for your help!

**Comment by jackm9367** [comment]
Hey @manishkumar-dsi - just wondering if you're still working on this one? If not, I am more than happy to take this one on!

**Comment by manishkumar-dsi** [comment]
Hi. Currently I am not working on this. Due to time constraints. Please go ahead and work on this. Thanks

**Comment by jackm9367** [comment]
Not a problem - thank you. 
@stnguyen90 - could you please update assignee? Cheers!

**Comment by eldadfux** [comment]
Updated 👍

**Comment by jackm9367** [comment]
I think this issue can be resolved without a PR - the application shows a browser popup (JS alert) before member deletion (a `data-confirm` attribute in the button input). Looks like this was implemented in PR #1654. 
I tried to interpolate the member's name into the alert, however string interpolation did not work for the `data-confirm` attribute and would therefore require a custom solution on top which would reduce the readability of the code and would not follow the template structure for the rest of the project.
